Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -95,10 +95,13 @@ public final class RootTabBarViewController: UITabBarController {
   public func switchToMessageThread(_ messageThread: MessageThread) {
     self.switchToProfile()
 
+    let isDash = AppEnvironment.current.config?.features["ios_backer_dashboard"] == .some(true)
+
     guard let profileNav = self.selectedViewController as? UINavigationController,
-      let profileVC = profileNav.viewControllers.first as? ProfileViewController else {
-        return
-    }
+      let profileVC = (isDash
+        ? profileNav.viewControllers.first as? BackerDashboardViewController
+        : profileNav.viewControllers.first as? ProfileViewController)
```

Comment:
is this `isDash` check necessary? seems like you can just do `profileVC = profileNav.viewControllers.first` since you don't really care if it's the old profile or the new dashboard.